test/data/plumbing-test-jms-dev.edn

```diff
@@ -1,35 +1,104 @@
 {:ptc.util.jms/Properties
  {:action "start"
+  :sender "sender"
   :payload
           {:Zamboni
-           {:workflow "IlluminaArraysWorkflow"
-            :workflowPriority 1005
-            :zamboniClientVersion "e531c5382d56d69bb72e42e44bc2672ab25fdbb6_2020-07-16T10:08:44-0400"}
+           {:workflow             "IlluminaArraysWorkflow"
+            :workflowPriority     1005
+            :zamboniClientVersion "358adbd5343d8889dccd76cf3c544b5e113c8413_2020-08-25T13:11:32-0400"}
            :workflow
-           {:analysisCloudVersion 1
-            :beadPoolManifestPath "/home/unix/ptc/data/arrays/metadata/HumanExome-12v1-1_A/HumanExome-12v1-1_A.bpm"
-            :callRateThreshold 0.98
-            :chipManifestPath "/home/unix/ptc/data/arrays/metadata/HumanExome-12v1-1_A/HumanExome-12v1-1_A.1.3.extended.csv"
-            :chipName "HumanExome-12v1-1_A"
-            :chipWellBarcode "7991775143_R01C01"
-            :clusterFilePath "/home/unix/ptc/data/arrays/metadata/HumanExome-12v1-1_A/HumanExomev1_1_CEPH_A.egt"
-            :collaboratorParticipantId "NA12878"
-            :collaboratorSampleId "SM-3S2IV"
-            :gender "Female"
-            :genderClusterFilePath "/home/unix/ptc/data/arrays/metadata/HumanExome-12v1-1_A/HumanExomev1_1_gender.egt"
-            :greenIDatPath "/home/unix/ptc/data/arrays/HumanExome-12v1-1_A/idats/7991775143_R01C01/7991775143_R01C01_Grn.idat"
-            :labBatch "ARRAY-CO-5799466"
-            :participantId "PT-97GM"
-            :productFamily "Whole Genome Genotyping"
-            :productName "Infinium Exome V1_A"
-            :productOrderId "PDO-15923"
-            :productPartNumber "P-WG-0094"
-            :productType "aou_array"
-            :redIDatPath "/home/unix/ptc/data/arrays/HumanExome-12v1-1_A/idats/7991775143_R01C01/7991775143_R01C01_Red.idat"
-            :regulatoryDesignation "RESEARCH_ONLY"
-            :researchProjectId "RP-45"
-            :sampleAlias "NA12878"
-            :sampleId "SM-3S2IV"
-            :sampleLsid "broadinstitute.org:bsp.dev.sample:NOTREAL.NA12878"
-            :zCallThresholdsPath "/home/unix/ptc/data/arrays/metadata/HumanExome-12v1-1_A/IBDPRISM_EX.egt.thresholds.txt"}}
-  :sender "picard-dev"}}
+           {:aggregationPendingQueue                     "aggregation.pending.queue"
+            :analysisCloudPath                           "gs://storage/pipeline/204339030085_R08C01/v3/"
+            :analysisCloudVersion                        3
+            :analysisDirectory                           "/v3"
+            :beadPoolManifestPath                        "/home/unix/ptc/data/arrays/metadata/HumanExome-12v1-1_A/HumanExome-12v1-1_A.bpm"
```

Comment:
It would be even better to have a test input that uses the AoU chip. That's something can be done later though.

Reply:
Oh that's a good point! The chip name & chipwell barcode is different so I'm not sure if this message would work.